charts/prometheus-ipmi-exporter/Chart.yaml

```
@@ -4,9 +4,9 @@ description: This is an IPMI exporter for Prometheus.
 
 type: application
 
-version: 0.2.0
+version: 0.3.0
```

Bump to patch version shall be fine.
```suggestion
version: 0.2.1
```